### ISSUE TITLE
Fuzzy Dates and Chart

### ISSUE THREAD

**Issue by badapplesdatabase** [opennorth/sfm-proxy#12]
If Organization A has a Command parent relationship with Organization B, with citations from 2010 to May 5, 2014 - two questions:

1) What happens when the user moves the timeline to May 6, 2014?

2) What happens when the user moves the timeline to June 4, 2010 - does the command relationship display with the “fuzzy” date?

Same question if the first citation was June 2010

**Comment by badapplesdatabase** [comment]
If Person A is Commander for Organization A, with citations from 2012 to May 5, 2014 - AND Person B is Commander for Organization A with only one citations (not start date) of July 7, 2014 - three questions:
1) What happens when the user moves the timeline to May 6, 2014?
2) What happens when the user moves the timeline to June 4, 2012 - does the command relationship display with the “fuzzy” date?
3) What happens when the user moves the timeline to July 8, 2014?

**Comment by jpmckinney** [comment]
I think these questions are answered by trying them in the UI?

**Comment by badapplesdatabase** [comment]
UI for Chart currently doesn't have timeline function to test.

There is also a data/methodology issue with this --- 

We could add an "Assume to Current Date? (Y/N)" column for parent relationships so that when a user scrolls past the date of last citation the command chart isn't blank --- however this raises a UI issue - I worry that the date of last citation is not as clear for the user in the Chart as it is with the Map. As of now there doesn't seem to be a way for the user to find the dates of the parent relationship anywhere, its not in the full dossier or in the Chart.

**Comment by sebastien** [comment]
If I remember correctly, the chart's current date is the one previously selected on the timeline. Sadly, I cannot test it (changes in the API?) http://ffctn.com/clients/on/sfm/beta/#country/ng/o/ade98da3-7826-4047-b867-25e1beeb9f25!date=2014-01-01&focus=chart shows no chart.

**Comment by jpmckinney** [comment]
I have a changelog here: https://github.com/opennorth/sfm-proxy/blob/master/CHANGELOG.md

**Comment by badapplesdatabase** [comment]
@sebastien I think it will make all of our lives easier if I bring any outstanding UI issues from the Google Doc into the Github and either create new issues/ integrate into existing threads.  It should take me a day or two to pull everything in.

**Comment by sebastien** [comment]
OK. Just to re-iterate, we will go through the changes once the API is stabilized, as we only have 1 iteration left.

**Comment by badapplesdatabase** [comment]
Right! My thought is by getting up everything in Github we can make sure we are all on the same page so that the iteration will capture everything.

**Comment by sebastien** [comment]
Yes, it's going to be much easier to have a centralized issue repository. Thanks!
